README.rst

```diff
@@ -11,88 +11,88 @@ more information about the backend service.
 
 aiven-client (``avn``) is the official command-line client for Aiven.
 
-::
-
-                        `'+;`         `'+;`
-    The Aiven Crab    '@@@#@@@`     '@@@#@@@`
-    ~~~~~~~~~~~~~~   #@.     #@.   @@.     #@.
-                     @: ,@@   @@   @: ,@@   @@
-                    ,@  @@@@@ :@  :@  @@@@@ .@
-                     @  #@@@. #@   @` #@@@` @@
-                     @@      `@#   @@      `@#
-                      @@#. :@@+     @@#. :@@#
-                       `+@@@'        `#@@@'
-               ,;:`                             ,;;.
-             @@@@@@#     .+@@@@@@@@@@@@@'.    `@@@@@@@
-            @@@@@#    @@@@@@@@@@@@@@@@@@@@@@+    @@@@@@
-             @@@   ;@@@@@@@@@@@@@@@@@@@@@@@@@@@`  `@@;
-              `  `@@@@@@@@@@@        ;@@@@@@@@@@@
-          `@@@  '@@@@@@@@@@@@@       @@@@@@@@@@@@@`  @@@
-         '@@@` .@@@@@@@@@@@@@@@    `@@@@@@@@@@@@@@@  @@@@`
-         @@@@  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  @@@@
-        '@@@@  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  @@@@
-        ,:::;  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  ,:::
-           :@  ,@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  #@
-           @@@  +@#+#@@@@@@@@@@@@@@@@@@@@@@@@@#+#@.  @@@
-           @@@@        '@@@@@@@@@@@@@@@@@@@.        @@@@
-           @@@  @@@@@@+  @@@@@@@@@@@@@@@@@  @@@@@@;  @@@
-           @@  @@@@@@@@@  @@@@@@@@@@@@@@@ `@@@@@@@@@  @+
-              @@@@@@@@@@@ :@@@@@@@@@@@@@  @@@@@@@@@@@ '
-             `@@@@@@@@@@@       ```      ,@@@@@@@@@@@
-             `@@@@@@   '@                :@:   @@@@@@
-              @@@@@:                           @@@@@@
-               @@@@@                           @@@@@
-                @@@@#                         @@@@'
-
-Platform requirements
-=====================
-
-Aiven Client has been tested and developed on Linux and Mac OS X systems.
-It is a Python program that works with Python 3.6 or newer versions.
-The only external dependency is Requests_ (and certifi_ on Windows/OSX).
+.. contents::
+
+
+.. _platform-requirements:
+
+Getting Started
+===============
+
+Requirements:
+
+*  Python 3.6 or later
+
+*  Requests_
+
+*  For Windows and OSX, certifi_ is also needed
 
 .. _`Requests`: http://www.python-requests.org/
 .. _`certifi`: https://certifi.io/
 
-Installation
-============
+.. _installation:
 
-From PyPI (Linux/OSX)::
+Install from PyPi
+-----------------
+
+Pypi installation is the recommended route for most users::
 
   $ python3 -m pip install aiven-client
 
-From PyPI (Windows)::
 
-  c:\> python3 -m pip install aiven-client
+Build an rpm Package
```

Comment:
I'm not 100% sure but it looks like RPM should be written capitalized, so `rpm -> RPM` (here and in all other places).

Reply:
Thanks, fixed it!